src/corehost/common/pal.h

```diff
@@ -245,9 +245,7 @@ namespace pal
     bool get_current_module(dll_t *mod);
     bool getenv(const char_t* name, string_t* recv);
     bool get_default_servicing_directory(string_t* recv);
-
-    //On Linux, there are no global locations
-    //On Windows there will be up to 2 global locations
```

Comment:
Even before this change Linux does have a global location, and now it also has the registered location.
I'm not sure we should have the full description in the header file - it's relatively complex as it stands right now...

Reply:
Oh, I read this comment as referring to `get_global_dotnet_dirs`, which still does not return any directories on Linux. That seemed wrong to me (in my mind global would include the default and registered location), but then I realized that it was not used for fxr lookup and only multi-level framework/sdk/store - hence my thinking the distinction was unclear.

I guess I just found that name (`get_global_dotnet_dirs`) confusing, since it doesn't quite match what I think of as 'global'.

Reply:
I added comments to the three functions which are confusing... The names are actually technically correct, but I understand the confusion. Maybe I'm just too "in this" to come up with better names... If you have better ideas please share...